packages/next-swc/crates/next-api/src/pages.rs

```diff
@@ -809,13 +814,15 @@ impl Endpoint for PageEndpoint {
     #[turbo_tasks::function]
     async fn write_to_disk(self: Vc<Self>) -> Result<Vc<WrittenEndpoint>> {
         let output = self.output();
-        let output_assets = output.output_assets();
+        // Must use self.output_assets() instead of output.output_assets() to make it a
+        // single operation
+        let output_assets = self.output_assets();
 
         let this = self.await?;
 
         this.pages_project
             .project()
-            .emit_all_output_assets(output_assets)
+            .emit_all_output_assets(Vc::cell(output_assets))
```

Comment:
Isn't it already a Vc?

Reply:
yes, but we want to have a Vc of the output_assets operations